.github/workflows/publish.yml

```diff
@@ -1,14 +1,16 @@
 name: Publish simulator archive
 
 on:
-  release:
-    types: [ published ]
+  push:
+    tags:
+      - '**'
```

Comment:
Is this meant to be double?

```suggestion
      - '*'
```

Reply:
Yes, otherwise tags with `/` in them will misteriously not run